sdk/appconfiguration/ci.mgmt.yml

```diff
@@ -21,9 +23,12 @@ pr:
       - release/*
   paths:
     include:
-      - sdk/appconfiguration/
-    exclude:
-      - sdk/appconfiguration/azure-data-appconfiguration
+      - sdk/appconfiguration/ci.mgmt.yml
+      - sdk/appconfiguration/pom.mgmt.xml
+      - sdk/appconfiguration/mgmt-v2019_02_01_preview
+      - sdk/appconfiguration/mgmt-v2019_10_01
+      - sdk/appconfiguration/mgmt-v2019_11_01_preview
+      - sdk/appconfiguration/mgmt-v2019_06_01
 
 stages:
   - template: ../../eng/pipelines/templates/stages/archetype-sdk-management.yml
```

Comment:
Should all pipelines include their template as a trigger path?

@weshaggard @mitchdenny

Reply:
I don't don't think we should trigger every pipeline if something in under the eng folder changes that will be overkill. Instead we will let the core pipeline handle that testing.